### ISSUE TITLE
Error after adding new scene

### ISSUE THREAD

**Issue by redgoalsuk** [interfaced/zombiebox#7]
Hi 

I am just testing out this framework and came across a problem when adding a new scene. I installed the boiler plate app and then created a new scene using

npx zb addScene test

After that I wanted to build the app for tizen which threw up the following error:

![Screenshot from 2020-04-23 15-22-48](https://user-images.githubusercontent.com/63847979/80110849-0cc41180-8577-11ea-9e70-d9c88f978b80.png)

**Comment by l1bbcsg** [comment]
Hello, thanks for your interest in ZombieBox!

The error message indicates that you're missing Tizen SDK build tools, they're either not installed or installed but not present in PATH (or `platforms.tizen.tizenToolsDir` config property).
See [README in Tizen platform](https://github.com/interfaced/zombiebox-platform-tizen/blob/master/docs/INDEX.md) for more details.

The warning message (in yellow) is not a fatal error, but it may be indicative of some problem.

**Comment by l1bbcsg** [comment]
Oh, I figured the warning. It seems there's a mistake in path to `AbstractCuteScene` in template used by `addScene`. 
Correct import should be: 
```js
import AbstractCuteScene from 'cutejs/layers/abstract-scene';
```
While addScene generates:
```js
import AbstractCuteScene from 'cutejs/layers/abstract-cute-scene';
```
You can change that manually in the scene that's already generated, I'll try to publish a fix for `zombiebox-extension-cutejs` soon.

**Comment by l1bbcsg** [comment]
This was fixed in `zombiebox-extension-cutejs` and published in version `1.2.1`. 
`zombiebox` package itself was not affected.
